src/content/documentation/develop/web-ext-command-reference.md

```diff
@@ -339,6 +340,18 @@ web-ext run --start-url www.mozilla.com --start-url developer.mozilla.org
 
 Environment variable: `$WEB_EXT_START_URL`
 
+#### `--watch-ignored`
```

Comment:
Nit, let's add an empty line after the header (for consistency with the convention used for other section in this file).